dependencies/src/main/java/com/google/cloud/tools/opensource/dependencies/DependencyGraphBuilder.java

```diff
@@ -154,7 +155,7 @@ private DependencyNode resolveCompileTimeDependencies(
 
   /**
    * Finds the full compile time, transitive dependency graph including duplicates, conflicting
-   * versions, and dependencies with 'provided' scope.
+   * versions, and dependencies with provided and optional scope.
```

Comment:
optional is not a scope. I call "optional flag".

Reply:
true, rewrote